packages/katello/rubygem-redhat_access/rubygem-redhat_access.spec

```diff
@@ -25,6 +27,7 @@ Requires: %{?scl_prefix}rubygem(redhat_access_lib) >= 1.1.5
 Requires: %{?scl_prefix}rubygem(angular-rails-templates) >= 0.0.4
 Requires: %{?scl_prefix}rubygem(foreman-tasks)
 Requires: %{?scl_prefix}rubygem(katello)
+Requires: %{?scl_prefix}rubygem(foreman_rh_cloud) >= 3.0.17
```

Comment:
Why? I don't see that dep in https://github.com/redhataccess/foreman-plugin/blob/master/redhat-access/redhat_access.gemspec

If that's intentional, please either add it to the gemspec, or place it outside the "generated dependencies" block

Reply:
Moved it out of the autogenerated block. I have added it manually anyway.